app/assets/stylesheets/admin/enterprises.css.scss

```diff
@@ -29,3 +29,10 @@ form[name="enterprise_form"] {
     }
   }
 }
+
+.admin-enterprises-index__admin-actions-divider {
+  margin-top: 1em;
+  margin-bottom: 1em;
+  height: 1px;
+  background-color: $admin-enterprises-index__admin-actions-divider-color;
```

Comment:
The new color you've defined here has the same value as the existing `$pale-blue` variable, so you can just use: `background-color: $pale-blue`.